tests/test_validators.py

```diff
@@ -19,14 +20,23 @@ def validate(self, value):
             max_length=self.max_length,
         )
 
+    def validate_subtask_price_value(self, value):
+        return validators.validate_integer(
+            field_name=self.field_name,
+            value=value,
+        )
```

Comment:
I believe this is unrelated to `ValidateVarcharTestCase`

Reply:
indeed, @kbeker please create a separate test case